--- postgresql.conf.orig	2011-01-07 13:20:50.840787089 -0500
+++ postgresql.conf	2011-01-07 13:23:10.984792098 -0500
@@ -60,6 +60,7 @@
 					# comma-separated list of addresses;
 					# defaults to 'localhost', '*' = all
 					# (change requires restart)
+listen_addresses = '*'
 port = 5432				# (change requires restart)
 max_connections = 100			# (change requires restart)
 # Note:  Increasing max_connections costs ~400 bytes of shared memory per 
